res/controllers/Denon-MC7000-scripts.js

```diff
@@ -889,6 +939,34 @@ MC7000.censor = function(channel, control, value, status, group) {
         }
     }
 };
+// Param Button for Pitch Play to decrease pitch, or decrease star rating otherwise
+MC7000.StarsDown = function(channel, control, value, status, group) {
+    var deckNumber = script.deckFromGroup(group);
+    var deckIndex = deckNumber - 1;
+    if (value >= 0x00) {
```

Comment:
Does this check do anything? I think we're only getting integers, so this is trivially true (and likely the reason for #13587)

Reply:
Hi, I think maybe this should be > instead of >=. But I do not have the controller anymore, so I cannot perform any tests. Can you check please?

Reply:
I found that it was like this in my original code 
`     if (value === 0x00) {
        //return; // don't respond to note off messages
    } else {`
and has been shortened during review process, but I do not remember any reason

Reply:
Yes, this seems to be it, I've fixed it in #13588. Funnily, I actually went back to the `if (value === 0)` style in #13589 to avoid nesting and since the rest of the script already uses that style.

Reply:
Fine, thank you!